util/registry/registry.go

```diff
@@ -30,6 +30,9 @@ func (r registry[T]) AddCtx(name string, factory factoryFunc[T]) {
 }
 
 func (r registry[T]) Get(name string) (factoryFunc[T], error) {
+	if name == "" {
```

Comment:
Wozu brauchts den hier? Bei DB Config ist das immer gegeben. Auch die Meldung ist misleading ;)

Reply:
Der trigget wenn du ein required Attribut nicht angibst.

Reply:
Das wird drunter ja schon abgefragt.

Reply:
Hier ein Beispiel. Ich lege ein Custom Device an, habe aber das Feld `maxcurrent` gar nicht definiert.

vorher
```
cannot create charger type 'custom': maxcurrent: invalid plugin type:
```

nachher
```
cannot create charger type 'custom': maxcurrent: field is missing
```

Reply:
Die erste Meldung is verwirrend und ich musste selbst im Code suchen was hier das eigentliche Problem ist.

Reply:
Fair, aber an der Stelle gibts keine Notion von einem „field“. Dann sollten wir die Meldung ändern: „empty type“.

Reply:
Die Funktion hier kennt auch keine Attribute. `missing type` trifft es exakt.

Reply:
`missing plugin source` find ich sprechender. Gerade weil das Wort `type` nichts ist, was in der Nutzereingabe fehlt.

Reply:
Gehört aber auch nicht an diese Stelle- die Typenregistry weiss nichts von Plugins und wird identisch auch für Fahrzeuge, Charger etc verwendet...

Reply:
Das gehört in plugin/config.go plugin()